ckan/config/routing.py

```diff
@@ -390,6 +390,7 @@ def make_map():
         m.connect('/user/logged_out_redirect', action='logged_out_page')
         m.connect('/user/reset', action='request_reset')
         m.connect('/user/me', action='me')
+        m.connect('/user/check_permissions', action='check_permissions')
```

Comment:
could you make this URL clearly private? Maybe something like `/private/api/bpa/user_permissions`

Reply:
Fixed.